### PR TITLE
chore(cvsb-19059): adding pagination to getbysearchterm method

### PR DESCRIPTION
**Pull request by cb-cs**
## (BE) Remaining pagination and query filtering on the Tech records microservice

Adding pagination to getBySearchTerm method
[link to ticket number](https://jira.dvsacloud.uk/browse/CVSB-19059)

## Checklist

- [ ] Code has been tested manually
- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/services/TechRecordsService.ts [hunk 1]
```diff
@@ -295,8 +293,8 @@ class TechRecordsService {
       if (isTrailerIdValid.error) {
         return Promise.reject({statusCode: 400, body: formatErrorMessage("TrailerId is invalid")});
       }
-      const trailerIdRecords: QueryOutput = await this.techRecordsDAO.getBySearchTerm(techRecord.trailerId, SEARCHCRITERIA.TRAILERID);
-      if (trailerIdRecords.Count! > 0) {
+      const trailerIdRecords: ITechRecordWrapper[] = await this.techRecordsDAO.getBySearchTerm(techRecord.trailerId, SEARCHCRITERIA.TRAILERID);
```
Comment (bristol-mike-jones) [inline src/services/TechRecordsService.ts:296]: 👍  Much better as a ITechRecordWrapper array - You can get rid of QueryOutput from the imports :)

Comment (cb-cs) [reply src/services/TechRecordsService.ts:295]: This has now been removed, please resolve if happy with change. Thanks

**Review by bristol-mike-jones — DISMISSED** [review]
LGTM

**Review by m-coslett — APPROVED** [review]

**Review by m-ujaffer — DISMISSED** [review]

**Review by Daniel-Searle — DISMISSED** [review]

**Review by m-coslett — DISMISSED** [review]
Reviewed by Craig as I made the changes, all OK

**Review by Daniel-Searle — APPROVED** [review]

**Review by m-coslett — APPROVED** [review]

**Comment by IuliaDVSA** [comment]
CVSB-19059 merged

**Pull request merged**